src/Controller/Api/GroupController.php

```diff
@@ -159,7 +159,7 @@ public function postGroupAction(Request $request)
      *  }
      * )
      *
-     * @param int     $id      Group identifier
+     * @param string  $id      Group identifier
      * @param Request $request A Symfony request
```

Comment:
```suggestion
     * @param Request $request Symfony request
```
I think we can avoid using `the` or `a` in field description. It's look better in documentation.

Reply:
I've updated the suggested descriptions at Swagger declarations. Let's these docblock description changes for a separate PR.